core/frontend/src/IModelApp.ts

```diff
@@ -6,9 +6,12 @@
  * @module IModelApp
  */
 
+// @ts-expect-error resolves from the lib/{cjs,esm} dir
```

Comment:
does correctly resolve the pkg.json at runtime and will not copy the package.json into the lib/esm or lib/cjs dir at compile time